src/pages/api/csvUpload.ts

```diff
@@ -133,11 +137,11 @@ export const faresTriangleDataMapper = (
                     const errors: ErrorInfo[] = [
                         { id: errorId, errorMessage: 'The selected file contains an invalid price' },
                     ];
-                    setCsvUploadAttributeAndRedirect(req, res, errors);
+                    setCsvUploadAttributeAndRedirect(req, res, errors, poundsOrPence);
                     return null;
                 }
 
-                if (Number(price) % 1 !== 0) {
+                if (isPence && Number(price) % 1 !== 0) {
```

Comment:
I feel like if we dont do a check here if they select pounds, to see if they have any values that are silly (e.g. 100 or 150) we should definitely do the piece of work soon after to show them some example prices.